test/device/test_device.py

```diff
@@ -14,6 +14,47 @@ def test_device_directly_created_must_be_given_io_connection(buffer_connection):
     assert dev.io_connection == buffer_connection
 
 
+def test_device_add_neighbour_device(buffer_connection):
+    from moler.device.unixlocal import UnixLocal
+
+    dev1 = UnixLocal(io_connection=buffer_connection)
+    dev2 = UnixLocal(io_connection=buffer_connection)
+    f_devices = dev1.get_neighbour_devices(device_type=UnixLocal)
```

Comment:
change all f_devices to neighbour_devices

Reply:
Done